<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddTrack_UI</class>
 <widget class="QDialog" name="AddTrack_UI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>611</width>
    <height>194</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0" colspan="4">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="name_label">
       <property name="text">
        <string>Track name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="track_name"/>
     </item>
    </layout>
   </item>
   <item row="0" column="3">
    <widget class="KComboBox" name="comboTracks"/>
   </item>
   <item row="1" column="0" colspan="4">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QRadioButton" name="video_track">
       <property name="text">
        <string>Video &amp;track</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="audio_track">
       <property name="text">
        <string>A&amp;udio track</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="av_track">
       <property name="text">
        <string>AV track</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="arec_track">
       <property name="text">
        <string>Au&amp;dio record track</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Insert</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="1">
    <widget class="QSpinBox" name="tracks_count">
     <property name="suffix">
      <string> track</string>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="value">
      <number>1</number>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="KComboBox" name="before_select">
     <item>
      <property name="text">
       <string>Above</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Under</string>
      </property>
     </item>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AddTrack_UI</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AddTrack_UI</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
